mirrord/layer/tests/http_mirroring.rs

```diff
@@ -79,6 +79,7 @@ async fn test_mirroring_with_http(
                     mirrord_protocol::outgoing::DaemonConnect {
                         connection_id: 0,
                         remote_address: std::net::SocketAddr::from(([10, 253, 155, 219], 58162)),
+                        local_address: std::net::SocketAddr::from(([10, 253, 155, 218], 58161)),
```

Comment:
I think local_address should be 1.1.1.1 to be more inline with what is returned in NewTcpConnection or is this irrelevant with this test?

Reply:
irrelevant (this is udp)